src/logger.ts

```diff
@@ -60,3 +66,120 @@ export function sendCrashResponse({
     callback();
   }
 }
+
+export function loggingHandlerAddExecutionContext() {
```

Comment:
this is great! Were you able to do any end to end testing on this to see the final output logs?

Reply:
Yes. For example, by`console.log('Hello log!');`, it logs
`{"message":"Hello log!\n","logging.googleapis.com/labels":{"execution_id":"5a80ltr2xe5s"}}`.

However it doesn't work perfectly. For example, `console.log()` always extends a `\n` at the end of the string, which will go to the `message` field as well.

 Similarly, for string with format, like `throw Error`, it outputs `{"message":"[Function: Error] {\n  stackTraceLimit: \u001b[33m10\u001b[39m,\n  prepareStackTrace: \u001b[90mundefined\u001b[39m,\n  domainThrown: \u001b[33mtrue\u001b[39m\n}\n","logging.googleapis.com/labels":{"execution_id":"5ae4layspp8k"},"severity":"ERROR"}`,
comparing to original log in console and [cloud log](https://screenshot.googleplex.com/BaNwXQr9wAuCJ73)
![image](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/assets/27962750/be0486fa-cd46-40ee-9bac-286386cdd6e6)


Personally I think it's ok as it conveys the same message. WDYT?

Reply:
Discussed offline we're okay with it.